src/helpers/files.js

```diff
@@ -139,14 +139,47 @@ const registerFileActionFallback = () => {
 
 }
 
+const newRichWorkspaceFileMenuPlugin = {
+	attach(menu) {
```

Comment:
Can we merge this into the existing FilesWorkspacePlugin below? There should be no need for two separate plugins

Reply:
The plugin below gets the FileList as argument for the attach method but we need the menu instead.

Reply:
Right ... sorry I mixed that up then 👍